msal_extensions/windows.py

```diff
@@ -157,26 +157,15 @@ def add(self, event, **kwargs):
             super(WindowsTokenCache, self).add(event, **kwargs)
             self._write()
 
-    def update_rt(self, rt_item, new_rt):
+    def modify(self, credential_type, old_entry, new_key_value_pairs=None):
         with CrossPlatLock(self._lock_location):
             if self._needs_refresh():
                 try:
                     self._read()
                 except IOError as exp:
                     if exp.errno != errno.ENOENT:
                         raise exp
-            super(WindowsTokenCache, self).update_rt(rt_item, new_rt)
-            self._write()
-
-    def remove_rt(self, rt_item):
-        with CrossPlatLock(self._lock_location):
-            if self._needs_refresh():
-                try:
-                    self._read()
-                except IOError as exp:
-                    if exp.errno != errno.ENOENT:
-                        raise exp
-            super(WindowsTokenCache, self).remove_rt(rt_item)
+            super(WindowsTokenCache, self).modify(credential_type, old_entry, new_key_value_pairs)
```

Comment:
Nitpick: The current implementation is funtionally correct. However, because that `new_key_value_pairs` is a [keyword argument](https://docs.python.org/2/tutorial/controlflow.html#keyword-arguments), we should better explicitly use it as so:

```python
    super(WindowsTokenCache, self).modify(credential_type, old_entry,
        new_key_value_pairs=new_key_value_pairs)
```
or again use the `**kwargs` pattern:
```python
def modify(self, credential_type, old_entry, **kwargs):
    ...
    super(...).modify(credential_type, old_entry, **kwargs)
```

That way, the kwargs would remain orderless, which is semantically good. Should this method in base class obtain yet another new keyword argument in the future but with different order, your code here won't be broken. (Well, upstream library should NOT introduce new keyword argument in different order because it would be a subtle breaking change, but in reality, it happens anyway - it happened once in MSAL Python and [bit me too](https://github.com/AzureAD/microsoft-authentication-library-for-python/commit/2d6c373c29384d07ee294cb3f9fdc2dfab7dd0f7#diff-9803a6c326f35bba15eaa6b33829a874R97). My bad.)

Reply:
Doesn't sound like a nit to me! This is good feedback :) fixed in 9664d6b